src/components/App/App.component.jsx

```diff
@@ -32,25 +34,8 @@ function App() {
 
   return (
     <BrowserRouter>
-      <AuthProvider>
-        <Layout>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            <Route exact path="/login">
-              <LoginPage />
-            </Route>
-            <Private exact path="/secret">
-              <SecretPage />
-            </Private>
-            <Route path="*">
-              <NotFound />
-            </Route>
-          </Switch>
-          <Fortune />
-        </Layout>
-      </AuthProvider>
+      <Header />
```

Comment:
We can put `Header` and `Body` inside `Layout`

Reply:
but body contains `Layout`.

Reply:
can you please give me more details?

Reply:
but wouldn't `Body` already loop with that? or do you also mean to redesign `Body` completely?